src/status_im/contexts/keycard/pin/enter/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.contexts.keycard.pin.enter.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.events-helper :as events-helper]
+            [status-im.contexts.keycard.pin.view :as keycard.pin]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [on-complete]} (rf/sub [:get-screen-params])]
+    [rn/view {:style {:padding-bottom 12 :flex 1}}
+     [quo/page-nav
+      {:icon-name :i/close
+       :on-press  events-helper/navigate-back}]
+     [quo/page-top {:title (i18n/label :t/enter-keycard-pin)}]
+     [rn/view {:style {:flex 1}}]
```

Comment:
Is this empty view being used as a spacing element? For example, is it being used to push the `keycard.pin/auth` view to the bottom of the screen?

Reply:
yes